src/Uno.UI/UI/Xaml/UIElement.skia.cs

```diff
@@ -368,6 +369,8 @@ internal virtual void OnArrangeVisual(Rect rect, Rect? clip)
 		partial void HideVisual()
 			=> Visual.IsVisible = false;
 
+		public Visual GetVisualInternal() => ElementCompositionPreview.GetElementVisual(this);
```

Comment:
This method should not be public - I think the `IVisualElement2` interface should be implemented explicitly, e.g. `IVisualElement2.GetVisualInternal()`